### PR TITLE
Change header Validators format from bytes to array of address

### PR DESCRIPTION
**Pull request by liam-lai**
```
>eth.getBlock(1)
  validators: ["xdc5058dfe24ef6b537b5bc47116a45f0428da182fa", "xdc888c073313b36cf03cf1f739f39443551ff12bbe", "xdcefea93e384a6ccaaf28e33790a2d1b2625bf964d"]
```

### CODE REVIEW

File: consensus/XDPoS/engines/engine_v2/verifyHeader.go [hunk 1]
```diff
@@ -111,8 +111,10 @@ func (x *XDPoS_v2) verifyHeader(chain consensus.ChainReader, header *types.Heade
 		if header.Validators.CurrentEpoch == nil || len(header.Validators.CurrentEpoch) == 0 {
 			return utils.ErrEmptyEpochSwitchValidators
 		}
-		if len(header.Validators.CurrentEpoch)%common.AddressLength != 0 {
-			return utils.ErrInvalidCheckpointSigners
+		for _, address := range header.Validators.CurrentEpoch {
+			if len(address)%common.AddressLength != 0 {
```
Comment (wgr523) [inline consensus/XDPoS/engines/engine_v2/verifyHeader.go]: is this necessary? `len(address)` always is `common.AddressLength`?

File: consensus/tests/engine_v2_tests/adaptor_test.go [hunk 2]
```diff
@@ -67,7 +67,7 @@ func TestAdaptorGetMasternodesFromCheckpointHeader(t *testing.T) {
 	masternodesV1 := adaptor.GetMasternodesFromCheckpointHeader(headerV1)
 	headerV2 := currentBlock.Header()
 	headerV2.Number.Add(blockchain.Config().XDPoS.V2.SwitchBlock, big.NewInt(1))
-	headerV2.Validators.CurrentEpoch = common.Hex2Bytes("0278c350152e15fa6ffc712a5a73d704ce73e2e103d9e17ae3ff2c6712e44e25b09ac5ee91f6c9ff065551f0dcac6f00cae11192d462db709be3758c")
+	headerV2.Validators.CurrentEpoch = []common.Address{}
```
Comment (wgr523) [inline consensus/tests/engine_v2_tests/adaptor_test.go]: do we need "0278c350152e15fa6ffc712a5a73d704ce73e2e103d9e17ae3ff2c6712e44e25b09ac5ee91f6c9ff065551f0dcac6f00cae11192d462db709be3758c" split into address?

**Review by wgr523 — APPROVED** [review]

File: consensus/XDPoS/engines/engine_v2/verifyHeader.go (hunk 1 above)
Comment (liam-lai) [reply consensus/XDPoS/engines/engine_v2/verifyHeader.go]: good point, this type is always common.address type

File: consensus/tests/engine_v2_tests/adaptor_test.go (hunk 2 above)
Comment (liam-lai) [reply consensus/tests/engine_v2_tests/adaptor_test.go]: yes we need, but I don't know what exactly addresses are. Just split into 3?
```
0278c350152e15fa6ffc712a5a73d704ce73e2e1
03d9e17ae3ff2c6712e44e25b09ac5ee91f6c9ff
065551f0dcac6f00cae11192d462db709be3758c
```

Comment (wgr523) [reply consensus/tests/engine_v2_tests/adaptor_test.go]: yes

> yes we need, but I don't know what exactly addresses are. Just split into 3?
> 
> ```
> 0278c350152e15fa6ffc712a5a73d704ce73e2e1
> 03d9e17ae3ff2c6712e44e25b09ac5ee91f6c9ff
> 065551f0dcac6f00cae11192d462db709be3758c
> ```

**Pull request merged**